src/UiElement/UiElementTrait.php

```diff
@@ -14,6 +14,8 @@
 namespace MonsieurBiz\SyliusRichEditorPlugin\UiElement;
 
 use Exception;
+use PHPUnit\Framework\Assert;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** It's probably not this Assert class that you want here‏